src/dbg/debugger.cpp

```diff
@@ -127,7 +128,7 @@ static DWORD WINAPI memMapThread(void* ptr)
             memMapThreadCounter = GetTickCount();
         }
 
-        Sleep(50);
+        Sleep(1000);

```

Comment:
Just a quick thing I wanted to change. We should give the `TaskThread` an identifier and then store/load their timeout in the `settings.ini` this way you can easily customize how heavily you care about GUI updates. Setting the default to `100` milliseconds gave me instant GUI updates when spamming buttons, but nice skips (as desired) while holding the button.

There is also a bug where the first GUI update would not be registered correctly (it didn't change the CIP). I did not figure out why this happened yet.